bukuserver/templates/bukuserver/bookmarks_list.html

```diff
@@ -10,12 +10,14 @@
   {{ super() }}
   {% if data %}
     {% set _random = url_for('bookmark.details_view', modal=True, id='random', url=return_url, **(request.args|flt)) %}
-    <li><a id="random" data-target="#fa_modal_window" data-toggle="modal" href="{{ _random }}">Random</a></li>
+    <li><a id="random" data-target="#fa_modal_window" data-toggle="modal" href="{{ _random }}">{{ _('Random') }}</a></li>
   {% endif %}
 {% endblock %}
 
 {% block tail %}
+  {{ buku.fix_translations('bookmarks') }}
```

Comment:
Due to how filter groups are implemented in Flask-Admin, they have to be fixed in a very specific way to support l10n (even though the data model itself would allow for it)